Sources/Tron/Tron.pb.swift

```diff
@@ -0,0 +1,4377 @@
+// DO NOT EDIT.
```

Comment:
Can we include the original protobuf file and a script to generate this instead?

Reply:
I've updated README.md to include the steps